missioncontrol/api/views.py

```diff
@@ -179,13 +179,18 @@ def measure(request):
                 'version': version,
                 'data': []
             }
+            print("Start is %s\n" % start)
```

Comment:
leftover `print`

Reply:
I was checking for validating that `start` is an integer and you have that roughly L118. Did you mean `val` instead of `interval` in that error string?

Reply:
Nvm, `val` is out of scope in that error string. But if `start` is the one that has the error it won't be displayed.